DESCRIPTION

```diff
@@ -19,14 +19,14 @@ License: MIT + file LICENSE
 URL: https://pillar.r-lib.org/, https://github.com/r-lib/pillar
 BugReports: https://github.com/r-lib/pillar/issues
 Imports: 
-    cli (>= 2.3.0),
+    cli (>= 3.2.0),
```

Comment:
Do we really want to update versions in `DESCRIPTION` just to satisfy snapshot tests?

Reply:
Reverted versions bumps.